### PR TITLE
Set Context in RunnableSequence & RunnableParallel

### PR DESCRIPTION
**Pull request by hinthornw**

### CODE REVIEW

File: libs/core/langchain_core/runnables/fallbacks.py [hunk 1]
```diff
@@ -172,9 +174,12 @@ def invoke(
             try:
                 if self.exception_key and last_error is not None:
                     input[self.exception_key] = last_error
-                output = runnable.invoke(
+                child_config = patch_config(config, callbacks=run_manager.get_child())
```
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/fallbacks.py:177]: Do we need similar changes in RunnableParallel? 

Is there any utility code that can be pulled out?

Comment (hinthornw) [reply libs/core/langchain_core/runnables/fallbacks.py:177]: Likely ya

Comment (hinthornw) [reply libs/core/langchain_core/runnables/fallbacks.py:177]: Maybe `patch_and_set_context()` or something

Comment (hinthornw) [reply libs/core/langchain_core/runnables/fallbacks.py:177]: Decided against for now

**Review by eyurtsev — APPROVED** [review]

**Pull request merged**